reinstall error

```
ied install
rm -rf node_modules
ied install
```

```Error: Cannot find module '/Users/me/workspace/lib/node_modules/.tmp/beb952e7-ad2e-4da1-9378-61b6547389ed/package.json'```

--- a/lib/cache.js
+++ b/lib/cache.js
@@ -16,7 +16,7 @@ function read (uid) {
 }
 
 function fetch (dest, uid, cb) {
-  var untar = tar.extract(dest)
+  var untar = tar.extract(dest, {strip: 1})
   read(uid).on('error', cb)
     .pipe(gunzip()).on('error', cb)
     .pipe(untar).on('error', cb)

